modules/integration_gcp-memorystore-redis/conf/03-memory-usage-ratio.yaml

```diff
@@ -0,0 +1,20 @@
+## Example
+module: gcp-memorystore-redis
+name: memory usage ratio
+
+value_unit: "%"
+transformation: ".min(over='1h')"
+
+signals:
+  signal:
+    metric: stats/memory/system_memory_usage_ratio
+    extrapolation: zero
+
+rules:
+  critical:
+    threshold: 90
+    comparator: ">"
+  major:
+    threshold: 80
```

Comment:
Just in case, in [docs about monitoring memorystore](https://cloud.google.com/memorystore/docs/redis/monitoring-instances#create-stackdriver-alert) they use a 50% threshold. But perhaps that's too low.

Observing a production platform with this terraform module would help to adjust detector thresholds.

Reply:
We decide to keep these thresholds